tests/parallel/signal.rs

```diff
@@ -14,7 +14,7 @@ impl Signal {
         // otherwise calls to `sum` will tend to be unnecessarily
         // synchronous.
         if stage > 0 {
-            let mut v = self.value.lock();
+            let mut v = self.value.lock().unwrap();
```

Comment:
I wonder if we should disable the signals in shuttle tests? The signals are to construct a very specific scenario, defeating the benefit of shuttle - testing many possible scenarios.